style.css

```diff
@@ -47,16 +47,39 @@ main {
   background-color: #f7f7f9;
 }
 
-.introduction {
-  background-image: url(./images/header-bg.png);
-  background-repeat: no-repeat;
+.mobile-nav {
```

Comment:
- [ ] Please ensure adding mouse interaction to the menu button as well as the close button when the mouse is hovered or is focused the cursor should be a pointer instead 🖱️